.travis.yml

```diff
@@ -40,9 +40,9 @@ after_success:
   - echo "NPM_TAG=$NPM_TAG"
   - >
     [ "${TRAVIS_BRANCH}" = "master" ] &&
-    npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
+    yarn version --no-git-tag-version --new-version $GIT_TAG &&
```

Comment:
> Seems like it's a good idea to make a new git tag? Tags are used by GitHub to track releases: https://github.com/meetup/meetup-web-platform/releases

Sorry, I'm just seeing this. So yarn will make it's own git tags, but i'm assuming we want to use our own defined tags which is what the  `--new-version $GIT_TAG` argument (should) do. 

I'm circling back to see why this is breaking the build, but also, @mattkime or anyone else --- any suggestions on how to test a change like this? Thanks!